tests/components/aemet/test_weather.py

```diff
@@ -58,9 +58,9 @@ async def test_aemet_weather(
     assert state.attributes.get(ATTR_WEATHER_HUMIDITY) == 99.0
     assert state.attributes.get(ATTR_WEATHER_PRESSURE) == 1004.4  # 100440.0 Pa -> hPa
     assert state.attributes.get(ATTR_WEATHER_TEMPERATURE) == -0.7
-    assert state.attributes.get(ATTR_WEATHER_WIND_BEARING) == 90.0
-    assert state.attributes.get(ATTR_WEATHER_WIND_GUST_SPEED) == 24.0
-    assert state.attributes.get(ATTR_WEATHER_WIND_SPEED) == 15.0  # 4.17 m/s -> km/h
+    assert state.attributes.get(ATTR_WEATHER_WIND_BEARING) == 122.0
```

Comment:
Please use [] here since we expect the key to be present

Reply:
If you want to fix them all in another PR we can merge that rather quickly

Reply:
Done in d0fa007ac2ceb5157b6dfdf9e631e20937b0e92e

Reply:
Replaced it for forecast.get() too in 2f03278f92893bc31111b5817ca2cb62c956606a